docs/docsite/rst/os_guide/windows_ssh.rst

```diff
@@ -0,0 +1,202 @@
+.. _windows_ssh:
+
+Windows SSH
+===========
+
+On newer Windows versions, you can use SSH to connect to a Windows host. This is an alternative connection option than :ref:`WinRM <windows_winrm>`.
+
+.. note::
+    While Ansible could use the SSH connection plugin with Windows nodes since Ansible 2.8, official support was only added in version 2.18.
+
+.. contents::
+   :local:
+
+
+SSH Setup
+---------
+
+Microsoft provides an OpenSSH implementation with Windows since Windows Server 2019 as a Windows capability. It can also be installed through an upstream package under `Win32-OpenSSH <https://github.com/PowerShell/Win32-OpenSSH>`_. Ansible officially only supports the OpenSSH implementation shipped with Windows and not the upstream package and must be a minimum version of ``7.9.0.0``. This effectively means official support starts with Windows Server 2022 as Server 2019 ships with ``7.7.2.1``. Using older Windows versions or the upstream package may work but is not supported.
+
+To install the OpenSSH feature on Windows Server 2022 and later, use the following PowerShell command:
+
+.. code-block:: powershell
+
+    Get-WindowsCapability -Name OpenSSH.Server* -Online |
```

Comment:
For a lot of these long code blocks you might find it useful to save them to the native file type and then reference them in the RST with the ` .. literalinclude::` directive.

This approach results in cleaner RST and means you can lint / check the native file more easily. For example, opening `fancypython.py` in most IDEs would automatically check the file for issues. If the content of that same Python file is embedded in RST, it's a lot more difficult to spot syntax errors etc. If users copy/paste code directly from the docs, they also get any code issues.

If you like, you can find an example of how to use literal includes in the getting started guide: https://github.com/ansible/ansible-documentation/blob/d38b6000c7bed9d8f429898439a50f6f5001352d/docs/docsite/rst/getting_started/get_started_playbook.rst?plain=1#L26

Reply:
Do we lint PowerShell files? I know the lexer works but I'm unsure if CI does a sanity check on them like Python files.

Reply:
No, we don't link PowerShell files in CI. I just meant on the authoring side.